qiskit_metal/qlibrary/core/base.py

```diff
@@ -109,12 +116,11 @@ class QComponent():
     """
     # pylint: disable=too-many-instance-attributes
 
-    default_options = Dict(
-        # Note: If something is added here, _gather_all_children_options(cls) needs to be changed.
-        # Intended for future use, for components that do not normally take pins as inputs
-        # to be able to have an input pin and be moved/rotated based on said input.
-        # pin_inputs = Dict()
-    )
+    default_options = Dict(pos_x='0.0um',
+                           pos_y='0.0um',
+                           orientation='0.0',
```

Comment:
It may be helpful to have have some description of orientation default_option.

`The direction of the termination. 0 degrees is +x, following a counter-clockwise rotation (eg. 90 is +y).`

Reply:
hi, yes, this is the description currently:
```
orientation: '0.0' -- The primary direction in degrees of the QComponent.
          Expressed counter-clockwise orientation.
```
I intentionally removed the rest of the text you cite because it is incorrect.

Reply:
Thanks - I see it on line 77.